kuma/spam/models.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
I prefer the habit of importing this as `_lazy`. Otherwise you always have to look at how `_` is aliased to know whether it is the lazy one or not.